appveyor.yml

```diff
@@ -1,5 +1,8 @@
 environment:
-  SCCACHE_DIGEST: f808afabb4a4eb1d7112bcb3fa6be03b61e93412890c88e177c667eb37f46353d7ec294e559b16f9f4b5e894f2185fe7670a0df15fd064889ecbd80f0c34166c
```

Comment:
This seems like an unintentional change?

Reply:
Oh sorry forgot to comment on this but it's intentional, this is long since not used at all